refs/files-backend.c

```diff
@@ -3402,6 +3402,18 @@ static int files_ref_store_remove_on_disk(struct ref_store *ref_store,
 	return ret;
 }
 
+static int files_fsck(struct ref_store *ref_store,
+		      struct fsck_refs_options *o)
+{
+	int ret = 0;
+
+	struct files_ref_store *refs =
+		files_downcast(ref_store, 0, "fsck");
```

Comment:
You want to have the `REF_STORE_READ` capability here when downcasting. Also, it is more customary to declare all variables in a single block and separate the declarations from code via a newline.

There is also no need to initialize `ret`.